src/link-handler.js

```diff
@@ -82,10 +82,10 @@ export class DefaultLinkHandler extends LinkHandler {
     info.anchor = target;
     info.href = href;
 
-    let hasModifierKey = (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey);
+    let leftButtonClicked = event.which ? event.which === 1 : event.button & 1;
```

Comment:
Got it, we can check here only for `event.which` which is supported by IE9+ and other browsers and will be equal to `1` for the main mouse button.

About checks for meta keys - there is the same check on [this line](https://github.com/aurelia/history-browser/blob/1a67ae11885db085d6921f78e0ca69afb616c883/src/link-handler.js#L77) and after that `hasModifierKey` will always be `false`. What do you think about it?